src/openrct2/interface/Window.cpp

```diff
@@ -6,6 +6,7 @@
  *
  * OpenRCT2 is licensed under the GNU General Public License version 3.
  *****************************************************************************/
+#pragma optimize("", off)
```

Comment:
What is the purpose of this? (I don't recognise this)

Reply:
It helps me debug the code without having to use Debug builds, its just disabled optimizations for this file, left it there by accident.